generated/nidcpower/_converters.py

```diff
@@ -4,6 +4,32 @@
 import six
 
 
+def _string_to_list(string, prefix):
```

Comment:
Rename: _repeated_capability_string_to_list()
it's a very specific string syntax...